tests/cdb2_admin.test/runit

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+
+[[ $debug == 1 ]] && set -x
+set -x
+dbname=$1
+cdb2_admin=${TESTBUILDDIR}/tools/cdb2_admin_connection
+iterations=10
+successful_conns=$($cdb2_admin -d $dbname -t "localhost" -i $iterations)
```

Comment:
It will likely fail in clustered mode where client and database run on separate machines.